src/team/data_preprocessing.py

```diff
@@ -87,23 +87,14 @@ def _align_data(self) -> None:
         for i, trajectory in enumerate(self.trajectories_to_align):
             if i == self.reference_index:
                 continue
-            _, paths = dtw_ndim.warping_paths(tcp_ref, trajectory.tcp)
-            # best matching transformation
-            path = dtw.best_path(paths)
-            aligned_path = [p[1] for p in path]
+            path = dtw_ndim.warping_path(
+                from_s=trajectory.tcp, to_s=tcp_ref, window=10, psi=2
```

Comment:
I see the computational advantage; however, I am a bit worried about using such a small window. With window=10 for example, we would consider only 10 points instead of the entire serie for finding the best matching and with a sampling frequency of 100 Hz, it would mean that we look in a window of 0.1 s. Sometimes, there can be few seconds of misalignment between series. Therefore, with such parameter, I believe that temporal misalignment larger than 0.1 s wouldn't be eliminated. What do you think? Is my interpretation correct?
Probably we can find a good balance with a larger window such as window=100

Reply:
That's possible, I think I misunderstood the meaning of the parameter. I thought `windows` was measured in the unit of the x axis and thus in seconds. Looking at the code, your interpretation is correct! Let me correct it to include 10 seconds of windows instead. Would that sound good?

Reply:
I think this should be resolved in the latest version

Reply:
Yes I believe that 10 seconds is enough :)